pkg/kubernetes/diff.go

```diff
@@ -12,6 +13,13 @@ import (
 
 // Diff takes the desired state and returns the differences from the cluster
 func (k *Kubernetes) Diff(state manifest.List, opts DiffOpts) (*string, error) {
+	// prevent https://github.com/kubernetes/kubernetes/issues/89762 until fixed
+	if k.ctl.Info().ClientVersion.Equal(semver.MustParse("1.18.0")) {
+		return nil, fmt.Errorf(`You seem to be using kubectl 1.18.0, which contains an unfixed issue
```

Comment:
It looks like [the fix](https://github.com/kubernetes/kubernetes/pull/89795) is merged, and staged in the 1.19 milestone.

Should we make this a blanket ban on 1.18.x unless we see it in a minor release?

Reply:
although there is a cherry pick into 1.18 https://github.com/kubernetes/kubernetes/pull/89827 so maybe it could be fixed sooner

Reply:
LGTM either way I guess. If they make another release with the same bug, I will be severely disappointed.

Reply:
Okay, we will leave it as is then